### PR TITLE
Make header translucent on scroll down

### LINKED ISSUE #2108: Make header translucent when the user scrolls down
Make header translucent when the user scrolls down. Follow the same design as https://fossasia.org/

### PR DESCRIPTION
**Pull request by notlmn**
### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

### Short description of what this resolves:
The PR makes the header translucent when the user scrolls down. The original motivation it to make it look similar to the FOSSASIA website, as mentioned in #2108. 

### Changes proposed in this pull request:
- The PR adds an additional class to the navbar, that is specifically used to get the DOM node of navbar inside JS.
- Adds JS that executes on the main event page (homepage), can be expanded by moving JS to `footer.hbs`, need suggestion on this!

----------------

Closes #2108

### CODE REVIEW

**Comment by geekyd** [comment]
@notlmn Put the js code inside the navbar.js(https://github.com/fossasia/open-event-webapp/blob/development/src/backend/assets/js/navbar.js) file.

Also it will be great if you could share a link of sample event(gh-pages of the generated event) here. It will help move things quickly. For event samples please check https://github.com/fossasia/open-event/tree/master/sample

**Comment by notlmn** [comment]
> @notlmn Put the js code inside the navbar.js(/src/backend/assets/js/navbar.js@development) file.

Done!

------------------

> Also it will be great if you could share a link of sample event

Published to https://notlmn-dump.github.io/opev-gen-googio using the GoogleIO17 sample!

-----------------

(PS: There maybe a problem with the header logo which is not caused by this PR, that may be another issue for #2106 to handle)

**Comment by shubham3211** [comment]
Great work. LGTM :+1: 
Please squash the commits.

**Comment by notlmn** [comment]
> Implement disappear on scroll up as on chat.susi.ai/overview as mentioned in #2085.

@shubham3211 I didn't want this PR to close @2085 because the last point in that issue is a bit unclear, I thought it better be another PR that implements hide-on-scroll and the other unclear point!

--------------

> Please squash the commits.

This could be done in GH UI too right? How would this be helful if I locally squash commits and force push to GH? I thing it would be too hard to follow the progress if squashed!

**Comment by notlmn** [comment]
:tada: Yay!

**Comment by shubham3211** [comment]
@notlmn I asked you to squash the commits because the third commit is not relevant, there is no need for anyone to know that you moved the code from event.hbs to navbar.js . 
Great Work keep contributing :+1: .
